task_03.py

```diff
@@ -0,0 +1,84 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+"""Task 03 Module"""
+
+import decimal
+
+CUSTNAME = str(raw_input('What is your name? '))
+PRINCIPAL = int(raw_input('What is the amount of your principal? '))
+LOANYEAR = int(raw_input('For how many years is this loan borrowed? '))
+PREQUAL = raw_input('Are you prequalified for this loan? ')
+RATE = 0
+TOTAL = 0
+LOANREPORT = ''
+
+if 0 < PRINCIPAL <= 199999:
```

Comment:
instead of closing with a big `else` consider just setting `RATE` at line 11 to `None` at that point